src/tests/helper-functions.js

```diff
@@ -0,0 +1,219 @@
+};
+
+/**
+ *
+ * A common function that will trigger a failure if the page fails accessibility checks.
+ *
+ * @param failures {Array} - An array of failure messages returned by an accessibility scanner (see below).
+ * @param shouldHaveFailures {Boolean} - Whether the results should contain failures (`false` by default).
+ */
+gpii.test.webdriver.checkAccessibilityScanResults = function (failures, shouldHaveFailures) {
+    if (shouldHaveFailures) {
+        jqUnit.assertTrue("There should be at least one accessibility check failure...", failures.length > 0);
+    }
+    else {
+        jqUnit.assertTrue("There should be no accessibility check failures...", failures.length === 0);
+        fluid.each(failures, function (failure) {
+            jqUnit.fail("An accessibility check failed:\n" + JSON.stringify(failure, null, 2));
+        });
+    }
+};
+
+/**
+ *
+ * A function that can invoke any function on the client side with the given arguments.  Both jQuery and Fluid must be
+ * loaded on the target page before you can use this function.
+ *
+ */
+gpii.test.webdriver.invokeGlobal = function (functionPath, fnArgs, environment) {
+    return fluid.invokeGlobalFunction(functionPath, fnArgs, environment);
+};
+
+fluid.registerNamespace("gpii.test.webdriver.axe");
+
+/**
+ *
+ * A function to run aXe in a browser and then return the results.  Should be used with `executeAsyncScript`.
+ *
+ * Requires the accessibility developer toolkit to have already been loaded (for example, by
+ * using `executeScript` to pass its source to the browser.
+ *
+ * @param callback {Function} - The WebDriver API itself supplies a callback that we use to return the results of the scan.
+ */
+gpii.test.webdriver.axe.runAxe = function () {
+    var callback = arguments[arguments.length - 1];
+
+    /* globals axe */
+    if (arguments.length > 1) {
+        axe.configure(arguments[0]);
+    }
+
+    // TODO: We cannot use fluid.invokeGlobal here because we need to use the browser's `document` variable.  Discuss with Antranig.
+    axe.a11yCheck(document, callback);
+};
+
+/**
+ *
+ * Trigger test failure(s) if there are any violations reported by aXe.
+ *
+ * @param results {Object} - The raw results returned by aXe.
+ * @param shouldHaveFailures {Boolean} - Whether there should be failures (set to `false` by default).
+ *
+ */
+gpii.test.webdriver.axe.checkResults = function (results, shouldHaveFailures) {
+    gpii.test.webdriver.checkAccessibilityScanResults(results.violations, shouldHaveFailures);
+};
+
+fluid.registerNamespace("gpii.test.webdriver.axs");
+
+/**
+ *
+ * A function to run the accessibility developer toolkit in a browser and then return the results.  Should be used
+ * with `executeScript`.
+ *
+ * Requires the accessibility developer toolkit to have already been loaded (for example, by
+ * using `executeScript` to pass its source to the browser.
+ *
+ */
+
+
+gpii.test.webdriver.axs.runAxs = function (axsOptions) {
+    /* globals axs */
```

Comment:
I'd prefer to see this moved up to the head of the file - given a substantial purpose of these definitions (as well as suppressing the lint warning) is to announce to the reader of a file that it contains global references